### ISSUE TITLE
SciPy deprecation warnings

### ISSUE THREAD

**Issue by dihm** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#97]
Incidentally turned off the default filtering of deprecation warnings and found that arc relies on a number of "soon" to be deprecated functions of scipy. 

The two I have explicitly found are:

```python
import warnings
# turn on reporting of DeprecationWarnings
warnings.filterwarnings("default")
import arc
atom = arc.Rubidium85()
atom.getQuantumDefect(50,0,0.5)
```
Returns:
```shell
~\Miniconda3\envs\arc\lib\site-packages\arc\alkali_atom_functions.py:731: DeprecationWarning: scipy.floor is deprecated and will be removed in SciPy 2.0.0, use numpy.floor instead
  modifiedRRcoef = self.quantumDefect[int(floor(s) + s + j - l)][l]
```

```python
import warnings
# turn on reporting of DeprecationWarnings
warnings.filterwarnings("default")
import arc
atom = arc.Rubidium85()
atom.getDipoleMatrixElement(5,1,1.5,0.5,50,2,2.5,0.5,0)
```
Returns
```shell
~\Miniconda3\envs\arc\lib\site-packages\arc\wigner.py:252: DeprecationWarning: scipy.sqrt is deprecated and will be removed in SciPy 2.0.0, use numpy.lib.scimath.sqrt instead
  return WignerReturn * sqrt(TriaCoeff(j1, j2, j3) * TriaCoeff(j1, J2, J3) \
```

In looking at the imports for this, noticed that `wigner.py` already imports `floor` and `sqrt` from numpy, but then clobbers them by reimporting them from scipy as well.
https://github.com/nikolasibalic/ARC-Alkali-Rydberg-Calculator/blob/5521d5c4e7346516ddcbba52e47152d4d3944f43/arc/wigner.py#L7-L9

None of this seems critical since SciPy 2.0.0 is probably a ways away, but it is easy to fix and I personally would appreciate it so my pytest logs aren't clogged up with thousands of Deprecation warnings. If you'd prefer, I can knock together a PR that attempts to remove them all.

**Comment by nikolaqm** [comment]
Hi David @dihm ,

Maybe since you already identify source of problem, and might likely find a few more like that now you have systematic check, it would be great if you make a fix and submit a PR?
It will be gratefully merged to main asap. :)

Best wishes,
Nikola

**Comment by nikolasibalic** [comment]
Hi David @dihm ,

Maybe since you already identify source of problem, and might likely find a few more like that now you have systematic check, it would be great if you make a fix and submit a PR?
It will be gratefully merged to main asap. :)

Best wishes,
Nikola

**Comment by nikolasibalic** [comment]
Thank you for contributing fix. Released now as v3.1.7.
